### PR TITLE
remove uncle cellbase

### PR DESCRIPTION
**Pull request by zhangsoledad**
- [ ] update RFC#0019

### CODE REVIEW

File: docs/data-structures.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Data Structures of Cell/Script/Transaction/Block
```
Comment (doitian) [inline docs/data-structures.md]: Pick the changes to RFC. We'll remove this file.

File: miner/src/block_assembler.rs [hunk 2]
```diff
@@ -489,18 +484,17 @@ mod tests {
             // uncles_count_limit,
         } = block_template;
 
-        let (cellbase_id, cellbase) = {
-            let CellbaseTemplate { hash, data, .. } = cellbase;
-            (hash, data)
+        let cellbase = {
+            let CellbaseTemplate { data, .. } = cellbase;
```
Comment (quake) [inline miner/src/block_assembler.rs]: Do we still need `CellbaseTemplate` after removing cellbase id from header? Can we add it to `Vec<TransactionTemplate>` ?

Comment (zhangsoledad) [reply miner/src/block_assembler.rs]:
```
pub struct TransactionTemplate {
    pub hash: H256,
    pub required: bool,
    pub cycles: Option<Cycle>,
    pub depends: Option<Vec<u32>>,
    pub data: Transaction, // temporary
}
```
`cellbase` has no depends, and it's required no doubt, so perfer keep it.

File: docs/data-structures.md (hunk 1 above)
Comment (doitian) [reply docs/data-structures.md]: Posted here https://github.com/nervosnetwork/rfcs/pull/91#pullrequestreview-223619365

**Comment by doitian** [comment]
bors retry

**Comment by doitian** [comment]
Need to fix the merged generated flatbuffers file.

**Comment by doitian** [comment]
> Need to fix the merged generated flatbuffers file.

Regenerate verifier: https://github.com/nervosnetwork/ckb/pull/409

**Pull request closed without merging**